frontend/device/kobo/powerd.lua

```diff
@@ -271,58 +244,6 @@ function KoboPowerD:getWarmth()
     end
 end
 
--- Sets fl_warmth according to current hour and max_warmth_hour
--- and starts background job if necessary.
-function KoboPowerD:calculateAutoWarmth()
-    local current_time = os.date("%H") + os.date("%M")/60
-    local max_hour = self.max_warmth_hour
-    local diff_time = max_hour - current_time
-    if diff_time < 0 then
-        diff_time = diff_time + 24
-    end
-    if diff_time < 12 then
-        -- We are before bedtime. Use a slower progression over 5h.
-        self.fl_warmth = math.max(20 * (5 - diff_time), 0)
-    elseif diff_time > 22 then
-        -- Keep warmth at maximum for two hours after bedtime.
-        self.fl_warmth = 100
-    else
-        -- Between 2-4h after bedtime, return to zero.
-        self.fl_warmth = math.max(100 - 50 * (22 - diff_time), 0)
-    end
-    self.fl_warmth = math.floor(self.fl_warmth + 0.5)
-    -- Make sure sysfs_light actually picks that new value up without an explicit setWarmth call...
-    -- This avoids having to bypass the ramp-up on resume w/ an explicit setWarmth call on devices where brightness & warmth
-    -- are linked (i.e., when there's no mixer) ;).
-    -- NOTE: A potentially saner solution would be to ditch the internal sysfs_light current_* values,
-    --       and just pass it a pointer to this powerd instance, so it has access to fl_warmth & hw_intensity.
-    --       It seems harmless enough for warmth, but brightness might be a little trickier because of the insanity
-    --       that is hw_intensity handling because we can't actually *read* the frontlight status...
-    --       (Technically, we could, on Mk. 7 devices, but we don't,
-    --       because this is already messy enough without piling on special cases.)
```

Comment:
I don't recall all the details about this mess, because I don't care about the feature at all, but this might be an issue again with the new plugin.

We'll see. (I *really* don't want to deal with this ever again, so, eh').